apps/payments/vue/package.json

```diff
@@ -10,8 +10,8 @@
         "type-check": "vue-tsc --noEmit"
     },
     "dependencies": {
-        "@crossmint/client-sdk-smart-wallet": "workspace:*",
         "@crossmint/client-sdk-base": "workspace:*",
+        "@crossmint/client-sdk-smart-wallet": "workspace:*",
         "@crossmint/client-sdk-vue-ui": "workspace:*",
         "vue": "3.3.4",
         "vue-router": "4.2.4"
```

Comment:
not sure why this file changed, but it's formatted with prettier how it's supposed to be. Must've been something went wrong when merging with main.